command.py

```diff
@@ -0,0 +1,45 @@
+import json
+import argparse
+import commands
+
+def parseInput():
+    parser=argparse.ArgumentParser()
+    parser.add_argument("-cmd","--command",type=str)
+    parser.add_argument("-sch","--schema",type=str)
+    parser.add_argument("-pk","--primaryKey",type=str)
+    parser.add_argument("-val","--value",type=str)  
+    parser.add_argument("-dp","--data_base",type=str)
+    parser.add_argument("-tb","--table",type=str)  
+    
+
+    return parser.parse_args() 
+
+
+if __name__=='__main__':
+    args=parseInput()
+    data= None
+    if(args.schema is not None):
+        data=json.loads(args.schema)
```

Comment:
Why do you parse the schema here ? Do you expect schema in each command line input ?